numpy/core/src/multiarray/nditer_pywrap.c

```diff
@@ -37,6 +37,7 @@ struct NewNpyArrayIterObject_tag {
     npy_intp *innerstrides, *innerloopsizeptr;
     char readflags[NPY_MAXARGS];
     char writeflags[NPY_MAXARGS];
+    char open;
```

Comment:
Not `npy_bool`?

Reply:
the other flags are char, I was just following suit